src/scripts/qa.ts

```diff
@@ -1,23 +1,13 @@
-import {
-  Chain,
-  ChainId,
-  ChainType,
-  HashflowApi,
-  validateChain,
-} from '@hashflow/taker-js';
-import { PriceLevel } from '@hashflow/taker-js/dist/types/common';
+import {Chain, ChainId, ChainType, HashflowApi, validateChain,} from '@hashflow/taker-js';
+import {PriceLevel} from '@hashflow/taker-js/dist/types/common';
 import BigNumber from 'bignumber.js';
-import { getSecretValue } from 'helpers/secrets';
+import {computeLevelsQuote} from 'helpers/levels';
```

Comment:
eslint should space this, wondering why it looks different here.

Reply:
I ran eslint --fix but it didn't do anything? Maybe something is wrong with the eslintrc

Reply:
yeah this repo probably doesn't have the same eslintrc